### ISSUE TITLE
Project-level indentation settings

### ISSUE THREAD

**Issue by michelf** [michelf/sim-daltonism#22]
The project should set its own indentation settings instead of relying on everyone having the same settings in Xcode, which is quite unrealistic. This will make it easier to contribute.

**Comment by nschonni** [comment]
I'd suggest adding https://editorconfig.org to handle the config

**Comment by michelf** [comment]
Xcode has a built-in feature for that: select the project file and open the File inspector and look at the Text settings. That is what I was thinking of using.

I assume .editorconfig would be useful if someone was using another editor. Perhaps it'd also work for setting a tab width in GitHub?

**Comment by nschonni** [comment]
Yes, GitHub will respect the settings when rendering tabs in the code views, and also when using the web editor
